vividus-plugin-web-app/src/main/java/org/vividus/ui/web/RuntimeEnvironment.java

```diff
@@ -16,19 +16,30 @@
 
 package org.vividus.ui.web;
 
-import javax.inject.Inject;
-
 import org.vividus.ui.web.action.IJavascriptActions;
 
 public class RuntimeEnvironment implements IRuntimeEnvironment
 {
-    @Inject private IJavascriptActions javascriptActions;
+    private IJavascriptActions javascriptActions;
```

Comment:
might be final now

Reply:
it can't be since it's used in `ThreadLocal` fields initialization